torch/_dynamo/variables/builder.py

```diff
@@ -1707,15 +1711,22 @@ def wrap_numpy_ndarray(self, value):
         # that there's not another great way to do this atm.
         # This creates the right graphargs, as well as registration for guards in tensor names and shape env.
         LazyVariableTracker.realize_all(VariableBuilder(self.tx, source)(tensor_value))
+        example_value = wrap_to_fake_tensor_and_record(
+            tensor_value,
+            tx=self.tx,
+            is_tensor=False,
```

Comment:
is_tensor is dermined by whether target_cls in (TensorVariable, TensorWithTFOverrideVariable). NumpyNdarraryVaraible is not in them so is_tensor = False. This PR keeps this behavior unchanged and only do code-refactoring.